exit guideline for users and group owners (#1411)
rug-cit-hpc/league-of-robots: roles/online_docs/templates/mkdocs/docs/exit-guidelines-for-general-users.md

# Exit guidelines for general users

These guidelines should be used in case you stop using the cluster.

> So you leave the cluster neat and orginized, and important data stored in the correct location.

Reviewer: ```suggestion
So you leave the cluster neat and organized, and important data stored in the correct location.
```

Thank you in advance for taking the time to clean up your data, very much appreciated.  

#### First some general remarks. 
TMP is for calculating, this is not backed-up. PRM is for storaging of important data, this is backed-up. We now also privide an archive option, for sleeping datasets.
As you might know by our [code of conduct](/../coc_umcg_research_clusters/), you are obliged to work according the UMCG research code.
According to the [UMCG research code](https://researchcode.umcgresearch.org/en/), you are obliged to have a [Data Management Plan (DMP)](https://umcg.zenya.work/portal/#/document/99b46c52-2c81-4193-b688-ad5a01d1a496?scope=global&searchText=data%20management%20plan) for your project(s).

> In this DMP, a lot of information gathered concerning your project, including retention time, and where you are supposed to store your data.

Reviewer: ```suggestion
In this DMP, a lot of information must be gathered concerning your project, including retention time, and where you are supposed to store your data.
```

In these guidelines a few easy steps to follow to clean out your part of the cluster. 

#### The steps to follow:
- Get your DMP(s).
- Go to your tmp folder on the cluster. Check all clusters, if you used more than 1.
- Gather all important data, data you need to store for publication/storage/FAIRdata purposes and organise it logicaly. Be critical, only keep the important data.

> - Write or adapt the README for each data set/project. 

Reviewer: ```suggestion
- Write or adapt the README for each dataset/project. 
```
